includes/admin/views/html-admin-page-addons.php

```diff
@@ -9,7 +9,7 @@
 	exit;
 }
 
-echo '<h1 class="screen-reader-text">' . esc_html__( 'WP Job Manager Add-ons', 'wp-job-manager' ) . '</h1>';
+echo '<h1 class="screen-reader-text">' . esc_html__( 'WP Job Manager Marketplace', 'wp-job-manager' ) . '</h1>';
```

Comment:
Should we simply this to "Marketplace", or maybe "Job Manager Marketplace"?

<img width="541" alt="image" src="https://github.com/Automattic/WP-Job-Manager/assets/176949/7f3b514d-13f6-4797-9c04-199896f1ffbb">

Reply:
What about "Job Manager Marketplace"?

Reply:
Renamed to "Marketplace" in aaff8c2. Thanks!